components/serializer.rst

```diff
@@ -573,6 +573,52 @@ having unique identifiers::
     echo $serializer->serialize($org, 'json');
     // {"name":"Les-Tilleuls.coop","members":[{"name":"K\u00e9vin", organization: "Les-Tilleuls.coop"}]}
 
+Handling Arrays
+---------------
+
+The serializer component is capable of handling arrays of objects as well.
+Serializing such an array works just like serializing a single object:
+
+.. code-block:: php
+
+    $person1 = new Acme\Person();
+    $person1->setName('foo');
+    $person1->setAge(99);
+    $person1->setSportsman(false);
+
+    $person2 = new Acme\Person();
+    $person2->setName('bar');
+    $person2->setAge(33);
+    $person2->setSportsman(true);
```

Comment:
I would love to make this code sample a bit less foobar-ish and not advertise the use of setters:

```php
$john = new Person('John', 99);
$john->setSportsman(false);

$jane = new Person('Jane', 33);
$jane->setSportsman(true); // is there a gender neutral word for sportsman? :)
```